modules/network/src/main/java/org/apache/ignite/internal/network/MulticastNodeFinder.java

```diff
@@ -291,7 +292,11 @@ public void start() {
                     throw new IgniteInternalException(INTERNAL_ERR, "Error in multicast listener", e);
                 }
             } finally {
-                sockets.forEach(MulticastSocket::close);
+                try {
+                    IgniteUtils.closeAll(sockets);
+                } catch (Exception e) {
+                    throw new IgniteInternalException(INTERNAL_ERR, "Could not close multicast sockets", e);
```

Comment:
On a second thought: we should not throw from a finally block, so probably it makes sense to just log the exception (with WARN) and don't rethrow it